src/amo/components/App/index.js

```diff
@@ -62,6 +63,8 @@ export function getErrorPage(status: number | null) {
       return NotAuthorizedPage;
     case 404:
       return NotFoundPage;
+    case 451:
```

Comment:
@willdurand I added this, but I'm not sure if we need it/if it will do anything. I can't actually figure out how this is used. When does the code here actually do anything, and how can we get it to fire? I failed to be able to do so in my local testing.

Reply:
Oh, is it for when the server responds with the status for the actual page request (as opposed to an API request)?

Reply:
I don't think we should change anything here. I think it's only used when the server code throws an error (but I don't remember very well).